### ISSUE TITLE
not possible to use autocomplete feature with screen-reader when Visual studio code loses focus and gains it again.

### ISSUE THREAD

**Issue by nick777999** [microsoft/vscode#53719]
- VSCode Version: 1.25.0
- OS Version: windows 10

Steps to Reproduce:

1. Launch a screen reader in this case NVDA,
2. Open other files or apps,
3. Launch Visual Studio Code,
4. Open the file to edit,
5. Start editing and try to use autocomplete feature,
6. Switch to other window with keyboard shortcut Alt+Tab,
7. Return to Visual Studio Code window,
8. Continue editing and try to use autocomplete feature.

Does this issue occur when all extensions are disabled?: Yes

**Comment by nick777999** [comment]
Hi! The issue has been fixed after an update. Thank you all who participated in this. Recently I discovered another one, which also relates here. The same problem occurs when I am trying to use keyboard command Windows+d or Windows+m, which is command to go to desktop. I have performed several tests to see what is happening. They are given below.
• VSCode Version: 1.25.1.
• OS Version: windows 10
Steps to Reproduce:
1. Launch a screen reader, in this case NVDA.,
2. open some other app or window,
3. Launch the visual studio code,
4. Try to use autocomplete,
5. press key combination Windows+d or Windows+m,
6. Move to the Visual studio code with key combination Alt+tab,
7. Try to use the autocomplete feature.
Does this issue occur when all extensions are disabled?: Yes

test 2.
Steps to Reproduce:
1. Launch a screen reader, in this case NVDA.,
2. open some other app or window,
3. Launch the visual studio code,
4. Try to use autocomplete,
5. Switch to another window with key combination Alt+tab,
6. Press the key combination Windows+d or Windows+m,
7. Switch to visual studio code with Alt+Tab,
8. Try to use autocomplete.
Does this issue occur when all extensions are disabled?: Yes.

test 3.
Steps to Reproduce:
1. Launch a screen reader, in this case NVDA.,
2. open some other app or window,
3. Launch the visual studio code,
4. Try to use autocomplete,
5. Switch to another window with alt+tab,
6. press windows+d or windows+m,
7. return to the same or another window with alt+tab,
8. Return to visual studio code with alt+tab,
9. try to use autocomplete.
Does this issue occur when all extensions are disabled?: Yes.
Thanks in advance.

**Comment by isidorn** [comment]
@nick777999 thanks for testing out accessibility in vscode.
Let me just get this clear, the issue seems to be that when you return back to vscode suggest box is hidden and you expect it to still be visible, correct?

**Comment by nick777999** [comment]
hi! Thank you for reaching out  and for the hard work you put in accessibility . Yup, that’s right. The issue here is that the window hardly gets focused.  to get this working, I press applications  key and then escape.  when I tried auto complete in this context,  and tried to find  suggesting box, I  saw it. It means that this box is visible. I guess it is just the focus problem.  previously, when this focus issue was fixed, this was fixed automatically. That’s what makes me think this way.

**Comment by isidorn** [comment]
This should be fixed with our usage of latest Electron.
@nick777999 please let me know if you still see this and we can reopen. Thanks
